std/exception.d

```diff
@@ -1478,6 +1478,34 @@ private bool isUnionAliasedImpl(T)(size_t offset)
         static assert( isUnionAliased!(S.A5, 1)); //a5.b1;
 }
 
+package string errnoString(int errno)
```

Comment:
I thought it works only for templates. Where can I read on this topic? Why compiler can't infer attributes without auto?